modules/appeals_api/app/controllers/concerns/appeals_api/openid_auth.rb

```diff
@@ -55,6 +57,11 @@ def validate_auth_token!
 
     private
 
+    def unsafe_mode?
+      Rails.env.development? && \
+        Settings.dig(:modules_appeals_api, :enable_unsafe_mode) && ENV['APPEALS_API_UNSAFE_MODE'].present?
```

Comment:
Where would this environment variable come from on deploy? I think we'd have to work with someone to make sure it's set on certain vsp environments.

Reply:
When we talked about this amongst the team ([here](https://jira.devops.va.gov/browse/API-27449)), our goal was to give ourselves some way to skip authentication while developing locally, but there was a lot of concern that the setting could bet turned on accidentally in the deployed environments. Requiring both a setting and an environment variable was an idea we settled on to make it extra difficult for this to happen

Reply:
Gotcha! Let's put some comments to that affect at the top of the method so folk without that context can know why it's checking things in triplicate :)

Reply:
I second @cilestin's comment about adding a comment to explain the purpose. 

Personally, I feel pretty confident with the dev environment + setting checks alone, but I also don't see the additional environment variable check as a bad thing for extra safety. (I also can't remember what my feedback was back when we discussed it as a team.) The only (small) downside that I see is that the developer hoping to skip the token validation step locally has to remember to set the config in two places.

Reply:
I went ahead and removed the check for the environment variable here - it now only relies on running in dev + having the setting